src/main/java/org/webshar/hrms/dao/RoleRepositoryCustom.java

```diff
@@ -0,0 +1,9 @@
+package org.webshar.hrms.dao;
+
+import java.util.List;
+import org.webshar.hrms.models.db.Role;
+
+public interface RoleRepositoryCustom
+{
+    List<Role> getRoleNameLike(String name);
+}
```

Comment:
This needs to be present under org.webshar.hrms.repository package as per naming convention practice